src/components/SidebarNavigationItem/SidebarNavigationTitle.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { SidebarNavigationTitle } from "@/components";
+
+export default {
+  component: SidebarNavigationTitle,
+  title: "Sidebar/NavigationTitle",
+  tags: ["sidebar", "navigation-item", "autodocs"],
+};
+
+export const Default = {
+  args: {
+    icon: "user",
+    selected: false,
+    children: <a href="/profile">Profile</a>,
```

Comment:
let's make this different from the navigation item and more meaningful, so that someone reading the doc can have a better understanding.

Right now looking at the doc one would see `Profile` as the navigation item and `Profile` again as the navigation title, which doesn't make the difference between the two too clear  

Let's call this `tables` and the navigation item something like `system-table`